rasa/core/training/interactive.py

```diff
@@ -1444,9 +1444,13 @@ def _print_help(skip_visualization: bool) -> None:
         raise
 
 
-def _serve_application(app, stories, skip_visualization):
+def _serve_application(app, stories, skip_visualization, server_args):
     """Start a core server and attach the interactive learning IO."""
 
+    if server_args.get("port"):
```

Comment:
otherwise the variable might not be defined when running the server
```suggestion
    port = None
    if server_args.get("port"):
```